keras_cv/visualization/colors.py

```diff
@@ -0,0 +1,151 @@
+colors = {
```

Comment:
@fchollet @qlzh727 opinion on supporting colors in this way?  I think this is a good way to start the visualization package without adding a dependency to any other library.

Reply:
For an extra opinion we could also check the quite popular:
https://matplotlib.org/3.5.1/tutorials/colors/colors.html